OutOfSchool/OutOfSchool.WebApi/Services/ExternalExportProviderService.cs

```diff
@@ -111,4 +118,40 @@ private WorkshopInfoBaseDto MapToInfoWorkshopDto(Workshop workshop)
             ? mapper.Map<WorkshopInfoBaseDto>(workshop)
             : mapper.Map<WorkshopInfoDto>(workshop);
     }
+
+    private async Task FillRatingsForProviders(List<ProviderInfoBaseDto> providersDTO)
+    {
+        var providerIds = providersDTO.Select(p => p.Id).ToList();
+        var averageRatings = await averageRatingService.GetByEntityIdsAsync(providerIds).ConfigureAwait(false);
+
+        foreach (var providerDto in providersDTO)
+        {
+            if (providerDto is ProviderInfoDto providerInfoDto)
+            {
```

Comment:
(Code style) Instead of if you could use `.OfType<T>`. It removes extra nesting, and make your intention more clear:
`foreach (var providerInfoDto in providersDTO.OfType<ProviderInfoDto>())`